test_fakeredis.py

```diff
@@ -163,19 +149,13 @@ def test_saving_unicode_type_as_key(self):
         self.assertEqual(self.redis.set(u'Ñandu', 'foo'), True)
         self.assertEqual(self.redis.get(u'Ñandu'), b'foo')
 
+    @pytest.importorskip('builtins.bytes', reason='future.types not available')
     def test_future_newbytes(self):
-        try:
-            from builtins import bytes
-        except ImportError:
-            pytest.skip('future.types not available')
         self.redis.set(bytes(b'\xc3\x91andu'), 'foo')
         self.assertEqual(self.redis.get(u'Ñandu'), b'foo')
```

Comment:
I'm not an expert on pytest, but I suspect that this change breaks the test. Firstly, the documentation for importorskip says that it takes a module, not a member of a module. It's also not documented to be a decorator. Finally, it probably still won't put `bytes` into the local scope.

I haven't tested it, but I think the solution is to put something like this into the body of the function (in place of the code you've removed).
```python
bytes = pytest.importorskip('builtins', reason='future.types not available').bytes
```

Reply:
You are right! `pytest.importorskip` can't be used as a decorator, because it returns the module it tries to import, when it's successful. Should be fixed now!